apps/site/lib/site_web/controllers/schedule/schedule_api.ex

```diff
@@ -61,9 +61,14 @@ defmodule SiteWeb.ScheduleController.ScheduleApi do
     |> Stream.map(fn {trip_id, service} -> {trip_id, fares_for_service(service)} end)
     |> Stream.map(fn {trip_id, service} -> {trip_id, duration_for_service(service)} end)
     |> Stream.map(fn {trip_id, service} -> {trip_id, formatted_time(service)} end)
+    |> Enum.map(fn {trip_id, service} -> {trip_id, route_pattern(service)} end)
```

Comment:
Can we use `Stream.map` to match the above? for the lazy eval.

Reply:
sure -- wasn't fully clear on the advantage -- will update